### PR TITLE
Closes #1466 Unit test for `print.duplicates()` which closes test coverage gap

### LINKED ISSUE #1466: General Issue: Increase Test Coverage for `duplicates.R`
### Background Information

We need to increase test coverage for our functions. The below screenshot has the function and gap in test coverage that we need to write additional unit tests for  `duplicates.R`

image

### Definition of Done

New tests written that covers the red gap area for duplicates.R.

Use devtools::test_coverage() to reproduce the report

### LINKED ISSUE #2054: General Issue: Remove `print.duplicates()`
### Background Information

image
image

Messaging in print function is redundant, instead of traditional deprecation process as there is no suitable substitution proposal is to remove, as it exists nowhere else in our code base as well, unlikely users are using either as this was primarily for development-utility

### Definition of Done

Remove `print.duplicates()` and from codebase

### PR DESCRIPTION
**Pull request by zdz2101**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by zdz2101** [comment]
image
image

File: tests/testthat/test-duplicates.R [hunk 1]
```diff
@@ -35,4 +39,10 @@ test_that("dataset of duplicate records can be accessed using `get_duplicates_da
   )
 
   expect_true(all(expected_ouput == get_duplicates_dataset()))
+
+
+  # This is where print.duplicates comes in play when run by the user
+  capture <- capture_output(get_duplicates_dataset(), print = TRUE)
+  expected_print_output <- "Duplicate records with respect to `USUBJID`."
+  expect_true(stringr::str_detect(capture, expected_print_output))
```
Comment (bundfussr) [inline tests/testthat/test-duplicates.R:47]: Should we use the snapshot functionality mentioned by Daniel? Then we would check the complete output of the `print()` function.

**Review by bundfussr — APPROVED** [review]

**Review by bms63 — APPROVED** [review]
Let's go!!
